README.md

```diff
@@ -137,7 +135,7 @@ Headings can be used in anywhere and can have their size adjusted to suit the co
 |----------|------|-------------|
 | type | string | Set to `heading`. |
 | id | string (unique) | Set this to a unique string to allow this item to be looked up using `Clay.getItemsById()` in your [custom function](#custom-function). |
-| appKey | string (unique) | The AppMessage key matching the `appKey` item defined in your `appinfo.json`.  Set this to a unique string to allow this item to be looked up using `Clay.getItemsByAppKey()` in your custom function. You must set this if you wish for the value of this item to be persisted after the user closes the config page. |
+| appKey | string (unique) | The AppMessage key matching the `appKey` item defined in your `package.json`.  Set this to a unique string to allow this item to be looked up using `Clay.getItemsByAppKey()` in your custom function. You must set this if you wish for the value of this item to be persisted after the user closes the config page. |
```

Comment:
Not to be confusing, but `appKeys` exist in appinfo.json, `messageKeys` exist in package.json, and other variations result in a build error.